app/components/UserProfile/SecondaryProfile.tsx

```diff
@@ -0,0 +1,34 @@
+import { Avatar } from "@mui/material";
+import React, { FC } from "react";
+
+interface SecondaryProfileProps {
+  name: string;
+  avatarSrc: string;
+  avatarHeight?: number;
+  avatarWidth?: number;
+  className?: string;
+  placeOrder?: string;
+}
+
+const SecondaryProfile: FC<SecondaryProfileProps> = ({
+  name,
+  avatarSrc,
+  avatarHeight,
+  avatarWidth,
+  className = "",
+  placeOrder,
+}) => {
+  return (
+    <div className={className}>
+      <Avatar
+        className={placeOrder}
+        alt={name}
+        src={avatarSrc}
+        sx={{ height: avatarHeight, width: avatarWidth }}
+      />
+      <h1>{name}</h1>
```

Comment:
Add Tailwind classes for h1 (font weight, size, color) for consistency.
```suggestion
      <h1 className="text-lg font-medium text-gray-900">{name}</h1>
```

Reply:
as it is a child property of div and div className is styled by component, so it's styled by dynamically.